### DISCUSSION TITLE
Values FPS indicator

### DISCUSSION THREAD

**Discussion by Card15Nod2** [mathoudebine/turing-smart-screen-python#430] [General]
Hello,

I have the FPS but the values are wrong.

An idea ?

**Comment by Card15Nod2** [comment]
Fonctionne avec : 

Adrenalin Edition 24.5.1
Version du pilote
23.40.33.01-240508a1-403196C-AMD-Software-Adrenalin-Edition
Version du pilote Windows AMD
31.0.24033.1003

Python 3.12

turing-smart-screen-python-3.4.0

Windows 10

I have the FPS with the good values.

**Reply by aStroNuts04** [reply]
hi have you found a solution? i am unable to get the fps to show up as it says my gpu isnt supported but ive already installed afterburner and rivatuner

**Reply by Card15Nod2** [reply]
For me Work perfectly with : 

Adrenalin Edition 24.5.1
Version du pilote
23.40.33.01-240508a1-403196C-AMD-Software-Adrenalin-Edition
Version du pilote Windows AMD
31.0.24033.1003
Python 3.12
turing-smart-screen-python-3.4.0
Windows 10

I don't have afterburner and rivatuner.
Start with a clean Windows and follow the tutorial on Youtube, it should work without problems.
